spra-api/shared/src/main/scala/net/wiringbits/spra/api/models/package.scala

```diff
@@ -18,4 +19,15 @@ package object models {
 
   case class ErrorResponse(error: String)
   implicit val errorResponseFormat: Format[ErrorResponse] = Json.format[ErrorResponse]
+
+  implicit def optionFormat[T](using formatter: Format[T]): Format[Option[T]] = Format[Option[T]](
```

Comment:
I'm not sure about this, please add a TODO note to revisit the topic later, in theory, nulls should be formatted as nulls.